server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/services/ProjectBoundaryService.java

```diff
@@ -23,6 +23,13 @@
 import java.util.Set;
 import java.util.UUID;
 
+import org.locationtech.jts.geom.MultiPolygon;
+import org.locationtech.jts.geom.Geometry;
```

Comment:
Please remove, we don't need these dependencies at this point

Reply:
removed